src/components/themeSwitch/ThemeSwitch.jsx

```diff
@@ -0,0 +1,50 @@
+import { FaRegSun } from "react-icons/fa6";
+import { IoMoonOutline } from "react-icons/io5";
+import styles from "./themeSwitch.module.css";
+import { useTheme } from "../../hooks/useTheme";
+
+const ThemeSwitch = () => {
+  const { darkMode, toggleTheme } = useTheme();
+
+  return (
+    <div className={styles.switchWrapper}>
+      <label className={styles.switchContainer} aria-label="Toggle dark mode">
```

Comment:
- Remove the aria-label from the label or the input (but not both).
- Recommended: keep the dynamic aria-label on the input only.